chore: docs and warnings (#581)
repository-service-tuf/repository-service-tuf-cli: docs/source/guide/index.rst

    │ task                              Task Management Commands                                                           │
    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

> RSTUF CLI configuration file
> ================

Reviewer: Oh, forgot that you need to change that as well:
```suggestion
RSTUF CLI configuration file
============================
```

    │ task                              Task Management Commands                                                           │
    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

> RSTUF CLI config

Reviewer: ```suggestion
RSTUF CLI configuration file
```

================

> ``rstuf`` reads the settings configuration from config file. See:

Reviewer: ```suggestion
``rstuf`` will try to read the settings configuration from a configuration file. See:
```

> ``--config/-c``, default: ``$HOME/.rstuf.yml``.

Reviewer: ```suggestion
``--config/-c``, default path to the configuration file is: ``$HOME/.rstuf.yml``.
```

> This file contains the following settings:
> 
> * ``SERVER`` - The Repository Service for TUF API URL (optional).
> 
> * ``REPOSITORIES`` - The TUF repositoreis used by ``rstuf artifact`` commands.

Reviewer: ```suggestion
In this file, the following optional settings can be configured:

* ``SERVER`` - The Repository Service for TUF API URL.

* ``REPOSITORIES`` - TUF repositories used by ``rstuf artifact`` commands.
```

  .. note::

>     This settings can be automatically generated/updated by ``rstuf artifact repository`` commands.

Reviewer: ```suggestion
   You can generate or update this setting automatically by using ``rstuf artifact repository`` commands.
```

  This setting is a list of repositories with the following fields:
  ``name``, ``trusted_root`` (base64), ``metadata_url``, ``artifacts_url``
  (bool), and ``hash_prefix``.

  Example:

  .. code:: yaml

    REPOSITORIES:
        myrepo:
            artifact_base_url: http://127.0.0.1:8081
            hash_prefix: false
            metadata_url: http://127.0.0.1:8080
            trusted_root: aHR0cDovLzEyNy4wLjAuMTo4MDgwLzEucm9vdC5qc29u

* ``DEFAULT_REPOSITORY`` - The default repository to be used by ``rstuf artifact`` commands.

  .. note::

>     This settings can be automatically generated/updated by ``rstuf artifact repository set``
>     command.

Reviewer: ```suggestion
       You can generate or update this setting automatically by using ``rstuf artifact repository`` commands.
```
